website/docs/static.md

```diff
@@ -81,6 +81,19 @@ which errors are reported.
 > a signature in a `# typed: false` file might introduce new type errors if it's
 > called from a `# typed: true` file.
 
+### Upgrading a file from `ignore` to any other sigil
+
+After changing the sigil of an `ignore`d file, you must run
+`srb rbi hidden-definitions`.
+
+The last time hidden definitions were generated, the file was `ignore`d, sorbet
+didn't load it, so anything in it was treated as "hidden" and defined in
+`sorbet/rbi/hidden-definitions/hidden.rbi`. Now that sorbet is allowed to load
+it, those definitions must be removed from `hidden.rbi`.
+
+If you don't regenerate `hidden.rbi`, you are likely to encounter
+[error 4010](https://srb.help/4010) (two definitions of the same method).
```

Comment:
static.md is a pretty high level doc (most people reading it don't know what a `typed:` sigil is at all.
On the other hand, this suggestion is really specific to RBI files and specific error messages.

What do you think about moving this suggestion to the error-reference.md suggestion for 4010?
I'm fine linking to the error-reference.md from static.md, but I'd like to keep static.md somewhat short and high level.

Reply:
> static.md is a pretty high level doc (most people reading it don't know what a typed: sigil is at all.

I like the idea of keeping it high-level, but currently `static.md` is _where_ sigils are introduced. Each is then described in depth. After this, one's next question is "How do I upgrade a sigil?". I've located this new section accordingly.

> What do you think about moving this suggestion to the error-reference.md suggestion for 4010?

What I've written is not just about fixing 4010 (retroactive), it's about how to correctly upgrade a sigil (proactive). They should link to each other though.

> .. I'd like to keep static.md somewhat short and high level.

Maybe `static.md` could cover sigils at a higher level than it currently does, and the in-depth sigil docs could move to a new file, `sigils.md`. The new file could contain the detailed description of each sigil, and the proposed instructions re: upgrading sigils.